Multiparty/README.md

```diff
@@ -0,0 +1,295 @@
+# Multiparty sample
+
+This sample application shows how to arrange videos and to toggle the mic and camera in multiparty calls using OpenTok React Native SDK.
+
+The app builds upon the Basic Video Chat sample app.
+
+## Setup
+
+1. Install the required node modules: `npm install`
+
+2. For iOS, install the Podfile's dependencies: `cd ios/ && pod install`
+
+3. In the App.js file, set the `apiKey`, `sessionId`, and `token` properties to your Vonage Video API key (project ID), a Vonage Video session ID, and a token for that session.
+
+For testing, you can use the [OpenTok playground](https://tokbox.com/developer/tools/playground/) to create sessions, publish streams to the session from a web client, and view the stream published from the client using the OpenTok React Native SDK. Open the playground in multiple tabs to publish multiple streams to the session (from the browser). And mute the computer's speaker to prevent audio feedback
+
+## Understanding the code
+
+The App.js file includes all of the code that uses the OpenTok React Native SDK.
+
+### Joining the call
+
+The App component is the landing view when our App is launched. It has a primary conditional `render()` function to start the video call.
+
+The `joinCall` property of the React state triggers different views. When the App is launched, a simple View with a "Join the Call" button is displayed. The button triggers a state change and toggles the `joinCall` value to show the more complex video call view.
+
+## The video chat interface
+
+The app includes event handlers for events dispatched by the `OTSession` object, such as the `sessionConnected`, `sessionDisconnected`, `streamCreated`, and `streamDestroyed` events. (See the docs for these events: [Session Events](https://github.com/opentok/opentok-react-native/blob/develop/docs/OTSession.md#events)).
+
+The video call view is composed of the following components:
+
+* Publisher. The publisher view displays the primary user’s video stream.
```

Comment:
Nit.  Was it intentional to delimit with a period?  Why not `Publisher: ` or `Publisher -- ` instead?